src/components/Dropdown/index.tsx

```diff
@@ -51,6 +52,7 @@ export default class Dropdown extends React.PureComponent<DropdownProps, {}> {
           onChanged={this.handleChanged}
           placeHolder={placeHolder}
           label={label}
+          onRenderCaretDown={this.getIcon}
```

Comment:
you can also add `calloutProps={{ className: 'y-dropdown--callout' }}`, so you can scope the list CSS rules to that parent